app/modules/sightings/resources.py

```diff
@@ -476,20 +480,26 @@ def patch(self, args, sighting):
                 new_version = rdata['result'].get('version', None)
                 if new_version is not None:
                     sighting.version = new_version
-                with db.session.begin():
-                    db.session.merge(sighting)
+                    context = api.commit_or_abort(
+                        db.session,
+                        default_error_message='Failed to update Sighting version.',
+                    )
+                    with context:
+                        db.session.merge(sighting)
+            return rdata
 
-        else:  # no edm
-            context = api.commit_or_abort(
-                db.session, default_error_message='Failed to update Sighting details.'
-            )
-            with context:
-                parameters.PatchSightingDetailsParameters.perform_patch(
-                    args, obj=sighting
-                )
-                db.session.merge(sighting)
-            rdata[id] = str(sighting.guid)
-        return rdata
+        # no EDM, so fall thru to regular houston-patching
+        context = api.commit_or_abort(
+            db.session, default_error_message='Failed to update Sighting details.'
+        )
+        with context:
+            parameters.PatchSightingDetailsParameters.perform_patch(args, obj=sighting)
+            db.session.merge(sighting)
+        # this mimics output format of edm-patching
+        return {
+            'success': True,
```

Comment:
What is the purpose of this success field? I've seen it elsewhere too. It seems just to be a less detailed duplication of the status code. EDM may have had this but I don't understand why houston should.

Reply:
it is replicating what all the calls in WBIA do.   i believe there are rare cases where you get a 200 but success=false, at least in wbia.  i have just generally been returning out of houston the response from edm, thus me echoing same behavior here.

Reply:
I am thinking that we should not expose this (I believe) non standard behaviour at the houston level. 
If houston is supposed to be a clean break from the past would it be better to remove this now?